docs/manual/index-checker.tex

```diff
@@ -322,7 +322,8 @@ \section{Array minimum lengths\label{index-minlen}}
 \end{center}
   \caption{The two type hierarchies for array types used by the Index
     Checker.  On the left is a type system for minimum array lengths.  On
-    the right is a type system for arrays of equal length.  Qualifiers
+    the right is a type system for arrays of equal length ("a" and "b" are
+    assumed to be valid arrays).  Qualifiers
```

Comment:
What's a valid array?  I think this could just be "assuming "a" and "b" are arrays"